phase_1/keycloak/README.md

```diff
@@ -0,0 +1,33 @@
+# Keycloak (Java) SBOM Generation
+
+This workflow illustrates creation of SPDX and CycloneDX SBOMs for a java application. [Keycloak](https://github.com/keycloak/keycloak) due to its wide use and complexity. The workflow is well commented and can be used as a reference implementation for your own projects.
+
+- [.github/workflows/phase_1_keycloak.yml](../../.github/workflows/phase_1_keycloak.yml)
+
+## Reference Impelementation Objectives
+
+- Create both SPDX and CycloneDX SBOMs using open source tools
+- Showcase the additional SBOM lifecycle steps
+- Provide sample tools and implementation for others to use
+
+![SBOM Lifecycle](../../assets/lifecycle.svg)
+
+- `Generate SBOM with Trivy` Job
+  - __Tool__
+    - [trivy](https://github.com/aquasecurity/trivy)
+  - __Notes__
+    - Analyze the keycloak source code using [trivy](https://github.com/aquasecurity/trivy). There are several great open source sbom generation tools, and this could easily be repleased with a tool of your choice.
```

Comment:
```suggestion
    - Analyze the keycloak source code using [trivy](https://github.com/aquasecurity/trivy). There are several great open source sbom generation tools, and this could easily be replaced with a tool of your choice.
```